src/components/custom-media-library/DeleteConfirmationModal.tsx

```diff
@@ -0,0 +1,119 @@
+import { CSSProperties, useCallback, useState } from "react";
+import ReactModal from "react-modal";
+import { useModal } from "react-modal-hook";
+import { noop, takeOneValue } from "../../utils";
+import { FormattedMessage } from "react-intl";
+
+// TODO: consider moving these styles to classes in the Style Guide
+const deleteButtonStyles: CSSProperties = { backgroundColor: "#E2574C" };
+const modalOverlayStyles: CSSProperties = { background: "rgba(34,34,34,.5)" };
+const modalContentStyles: CSSProperties = { padding: 20 };
```

Comment:
@GusBaamonde these are my customizations for the confirmation popup.

Do you think that is there a way to move them to the style guide reusing current styles here?